rpcs3/Emu/RSX/VK/VKFragmentProgram.cpp

```diff
@@ -68,6 +68,12 @@ void VKFragmentDecompilerThread::insertIntputs(std::stringstream & OS)
 			OS << "layout(location=" << reg.reg_location << ") in " << PT.type << " " << var_name << ";" << std::endl;
 		}
 	}
+	
+	if (m_prog.front_back_color_enabled)
+	{
+		OS << "layout(location=11) in vec4 front_diff_color;";
+		OS << "layout(location=12) in vec4 front_spec_color;";
+	}
```

Comment:
Yes, i can see the table here .

https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/RSX/VK/VKCommonDecompiler.cpp#L232